neanderthal/src/main/kotlin/au/com/outware/neanderthal/presentation/view/activity/VariantListActivity.kt

```diff
@@ -85,6 +88,21 @@ class VariantListActivity : AppCompatActivity(), VariantListPresenter.ViewSurfac
         navigate(EditVariantActivity::class, EditVariantPresenter.ViewSurface.EXTRA_NAME to name)
     }
 
+    override fun goToMainApplication() {
+        val resolvedInfos = packageManager.queryIntentActivities(Intent(Intent.ACTION_MAIN), PackageManager.GET_RESOLVED_FILTER)
```

Comment:
https://developer.android.com/reference/android/content/pm/PackageManager.html#getLaunchIntentForPackage(java.lang.String)

This looks slightly more suited for purpose. Querying every MAIN activity on the device seems like it would be unnecessary; I'd be concerned that the checks in this for-loop are insufficient.